packages/suite/src/actions/suite/trezorConnectActions.ts

```diff
@@ -60,9 +63,10 @@ export const init = () => async (dispatch: Dispatch, getState: GetState) => {
     });
 
     try {
-        const connectSrc = isDesktop()
-            ? resolveStaticPath('connect/')
-            : 'https://connect.trezor.io/8/';
+        // @ts-ignore - Cannot find name window (suite-native)
+        const isTor = isTorDomain(window.location.hostname);
```

Comment:
i dont like this window reference here, maybe it would be better to put in in some `env` or `dom` util instead?

Reply:
ff8ec9b